packages/table/src/common/_loading.scss

```diff
@@ -14,25 +14,39 @@
   }
 }
 
-.bp-table-cell,
-.bp-table-header {
-  &.pt-loading {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    cursor: default;
-
-    // Always set color to transparent so cell content can fade-in when transitioning from the
-    // loading state
-    // stylelint-disable-next-line declaration-no-important
-    color: transparent !important;
-    pointer-events: none;
-  }
+@mixin hide-text() {
+  // Always set color to transparent so cell content can fade-in when transitioning from the
+  // loading state
+  // stylelint-disable-next-line declaration-no-important
+  color: transparent !important;
+}
+
+@mixin disable-interaction() {
+  cursor: default;
+  pointer-events: none;
+}
+
+@mixin vertical-align() {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
 
+@mixin table-skeleton($skeleton-height) {
   .pt-skeleton {
     opacity: 0;
-    height: $pt-grid-size / 2;
+    height: $skeleton-height * 1px;
```

Comment:
_whoa_ please just expect a unit from the variable. if the variable is called `height` then it should be usable as a `height` (ie, has a unit).